packages/terra-clinical-onset-picker/src/OnsetPicker.jsx

```diff
@@ -20,9 +20,15 @@ const GranularityOptions = {
   DATE: 'date',
 };
 
+const DATE_FORMAT = 'YYYY-MM-DD';
+
 const propTypes = {
   /**
-   * The ISO 8601 string representation of the birth date to calculate an onset date for the 'age' precision.
+   * The date unit of the age value. One of 'weeks', 'months', or 'years'.
+   */
+  ageUnit: PropTypes.string,
```

Comment:
Could use PropTypes.oneOf(['weeks', 'months', 'years']).

Reply:
Updated to ['weeks', 'months', 'years']